R/result.R

```diff
@@ -23,8 +23,11 @@ result <- function(x, message = NULL, correct = FALSE) {
 }
 
 
-# TODO-document
-# Should be what EVERY check_* returns
+#' \code{grader_graded} object
+#' returned by every \code{check_*} function
+#'
+#' @param message A character string of the message to be displayed.
+#' @param correct A boolean value of whether or not the checked code is correct.
 #' @export
 graded <- function(message, correct) {
```

Comment:
I know I originally wrote it as `message`, then `correct`.  

Can we switch it to `correct, message = NULL`?  (and any other code paths that need it fixed as well.